<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisColorSelectorSettings</class>
 <widget class="QDialog" name="KisColorSelectorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Color Selector</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Shape</string>
         </property>
         <property name="buddy">
          <cstring>comboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_2</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Color Harmonies</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_3</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Background Color</string>
         </property>
         <property name="buddy">
          <cstring>kcolorbutton</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KColorButton" name="kcolorbutton"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>etc</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0">
          <double>42.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Shade Selector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="buddy">
            <cstring>shadeSelectorType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="shadeSelectorType">
           <item>
            <property name="text">
             <string>MyPaint</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>When the widget gets shrunken</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shadeSelectorHideable">
         <property name="text">
          <string>hide shade selector</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allowHorizontalLayout">
         <property name="text">
          <string>change to a horizontal layout</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>do nothing</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Color Patches</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Last Used Colors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="lastUsedColorsShow">
            <property name="text">
             <string>Show</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Direction:</string>
                 </property>
                 <property name="buddy">
                  <cstring>lastUsedColorsAlignVertical</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="lastUsedColorsAlignVertical">
                 <property name="text">
                  <string>Vertical</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="lastUsedColorsAlignHorizontal">
                 <property name="text">
                  <string>Horizontal</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="lastUsedColorsAllowScrolling">
               <property name="text">
                <string>Allow Scrolling</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="lbl_lastUsedNumCols">
                  <property name="text">
                   <string>Number of columns</string>
                  </property>
                  <property name="buddy">
                   <cstring>lastUsedColorsNumCols</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="lastUsedColorsNumCols">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="lbl_lastUsedNumRows">
                  <property name="text">
                   <string>Number of rows</string>
                  </property>
                  <property name="buddy">
                   <cstring>lastUsedColorsNumRows</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="lastUsedColorsNumRows">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Common Colors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="commonColorsShow">
            <property name="text">
             <string>Show</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Direction:</string>
                 </property>
                 <property name="buddy">
                  <cstring>commonColorsAlignHorizontal</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="commonColorsAlignVertical">
                 <property name="text">
                  <string>Vertical</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="commonColorsAlignHorizontal">
                 <property name="text">
                  <string>Horizontal</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="commonColorsAllowScrolling">
               <property name="text">
                <string>Allow Scrolling</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="lbl_commonColorsNumCols">
                  <property name="text">
                   <string>Number of columns</string>
                  </property>
                  <property name="buddy">
                   <cstring>commonColorsNumCols</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="commonColorsNumCols">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="lbl_commonColorsNumRows">
                  <property name="text">
                   <string>Number of rows</string>
                  </property>
                  <property name="buddy">
                   <cstring>commonColorsNumRows</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="commonColorsNumRows">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>20</number>
                  </property>
                  <property name="value">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TODO: add this to the krita options dialog</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KisColorSelectorSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KisColorSelectorSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_commonColorsNumCols</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>commonColorsNumCols</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_commonColorsNumRows</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>commonColorsNumRows</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_commonColorsNumCols</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>commonColorsNumCols</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_commonColorsNumRows</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>commonColorsNumRows</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_lastUsedNumCols</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lastUsedColorsNumCols</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_lastUsedNumRows</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignVertical</sender>
   <signal>toggled(bool)</signal>
   <receiver>lastUsedColorsNumRows</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_lastUsedNumCols</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lastUsedColorsNumCols</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbl_lastUsedNumRows</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAlignHorizontal</sender>
   <signal>toggled(bool)</signal>
   <receiver>lastUsedColorsNumRows</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsShow</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsShow</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonColorsAllowScrolling</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastUsedColorsAllowScrolling</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
